src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -38,6 +38,13 @@ object TezosDatabaseOperations {
       )
     )
 
+
+  def writeFeesToDatabase(fee: List[Option[Fees]], dbHandle: Database): Future[Unit] =
```

Comment:
Please document this function. Also, let's use `averageFees` consistently everywhere as just 'fees' could mean anything.